cfgov/unprocessed/js/molecules/Autocomplete.js

```diff
@@ -45,9 +52,21 @@ function Autocomplete( element, opts ) {
   // Settings
   const _settings = {
     minChars: 2,
+
+    /* TODO: Is this sufficient to handle autocompletes without a maxlength?
+       https://developer.mozilla.org/en-US/docs/Web/API/Element/getAttribute#non-existing_attributes */
+    // 1024 is our upper limit for Elasticsearch queries
+    maxChars: _input.getAttribute( 'maxlength' ) ?
+      _input.getAttribute( 'maxlength' ) :
+      1024,
     delay: 300,
     url: '',
     list: [],
+
+    /* TODO: Should toggling the error message on and off be handled here in the
```

Comment:
I'd move it to ask-autocomplete.js since that's where the dom reference is and you now are broadcasting an event that can be listened to.

Reply:
Yeah, the more I think about this, the more I think it makes sense to move the error message stuff into the Ask organism. That keeps the underlying autocomplete molecule focused only on the input field and the results list. I made that change in 8b4c07f.